pkg/policy/selectorcache.go

```diff
@@ -172,6 +172,8 @@ type SelectorCache struct {
 
 	// map key is the string representation of the selector being cached.
 	selectors map[string]identitySelector
+
+	pop identityPopulator
```

Comment:
please give a better name to this field.

Reply:
Done!